test/conf/protractor.saucelabs.conf.js

```diff
@@ -1,19 +1,24 @@
-const {join} = require('path');
-let {config} = require('./protractor.shared.conf');
-const basicSpecs = '../basics.spec.ts';
-const saveMethodFolderSpecs = '../saveMethodsFolders.spec.ts';
-const checkMethodFolderSpecs = '../checkMethodsFolders.spec.ts';
-const deskSpecs = '../desktop.spec.ts';
-const mobileSpecs = '../mobile.spec.ts';
-const screenResolution = '1600x1200';
-const defaultCapabilities = {
+const { resolve } = require('path');
+const { config } = require('./protractor.shared.conf');
+
+const SCREEN_RESOLUTON = '1600x1200';
+
+const {
+	IC_SAUCE_USERNAME,
+	SAUCE_USERNAME,
+	IC_SAUCE_ACCESS_KEY,
+	SAUCE_ACCESS_KEY,
+	TRAVIS_JOB_NUMBER = 'local-build'
+} = process.env;
+
+const DEFAULT_CAPABILITIES = {
 	shardTestFiles: true,
-	tunnelIdentifier: process.env.TRAVIS_JOB_NUMBER,
+	tunnelIdentifier: TRAVIS_JOB_NUMBER,
```

Comment:
this now breaks testing on Sauce from a local machine. Previously this was undefined so a tunnel would never be started from a local machine. I need to check it but I think we can even leave out the `tunnelIdentifier` on Travis so this one will be obsolete